### ISSUE TITLE
Support `use server` directive for Server Actions

### ISSUE THREAD

**Issue by karlhorky** [daidodo/format-imports#23]
**Is your feature request related to a problem? Please describe.**

[Next.js 13.4 introduced Server Actions](https://nextjs.org/blog/next-13-4#server-actions-alpha), which require a new directive `'use server'`.

**Describe the solution you'd like**

This new `'use server'` directive should also be formatted in a similar way to how `'use client'` or `'use strict'` directives are formatted:

https://github.com/daidodo/format-imports/blob/db5d42e419446b442595d3ed631fad47db93382d/src/lib/format/parser/parser.ts#L143

This is what a `'use server'` directive looks like at the top of a file:

```js
'use server'; // Should be at top of file to indicate that this file should only ever run on the server

// more code...
```

One thing to keep in mind is that `'use server'` can also be allowed in a function instead of at the top of a file:

```tsx
// app/post/[id]/page.tsx

import kv from './kv';

export default function Page({ params }) {
  async function increment() {
    'use server';
    await kv.incr(`post:id:${params.id}`);
  }

  return (
    <form action={increment}>
      <button type="submit">Like</button>
    </form>
  );
}
```

**Describe alternatives you've considered**

Not formatting things and having a problem with Server Actions.

**Additional context**

`'use client'` was added for React 18 in this PR:

- https://github.com/daidodo/format-imports/pull/16

**Comment by daidodo** [comment]
Thanks for the feedback!

Yes, the directive can be handled the same way as `use client`. And the in-function directive is not a problem because the parser cares about top level statements only.

All suggestions and PRs will be welcomed!

**Comment by DerTimonius** [comment]
Hey everyone, I would like to take a look into this 🙂

**Comment by DerTimonius** [comment]
Just opened a PR here #24!

**Comment by daidodo** [comment]
Published v7.6.3 to include the feature.
